components/transport-netty-axis2-bridge/src/main/java/org/wso2/transports/http/bridge/listener/RequestUtils.java

```diff
@@ -176,17 +179,17 @@ static String getRestUrlPostfix(String uri, String servicePath) {
 
     static boolean isRESTRequest(String contentType) {
         return contentType != null &&
-                (contentType.contains("application/xml") ||
-                        contentType.contains("application/x-www-form-urlencoded") ||
-                        contentType.contains("multipart/form-data") ||
-                        contentType.contains("application/json") ||
-                        contentType.contains("application/jwt"));
+               (contentType.contains("application/xml") ||
+                contentType.contains("application/x-www-form-urlencoded") ||
+                contentType.contains("multipart/form-data") ||
+                contentType.contains("application/json") ||
+                contentType.contains("application/jwt"));
     }
 
     static boolean isRest(String contentType) {
         return contentType != null &&
-                !contentType.contains(SOAP11Constants.SOAP_11_CONTENT_TYPE) &&
-                !contentType.contains(SOAP12Constants.SOAP_12_CONTENT_TYPE);
+               !contentType.contains(SOAP11Constants.SOAP_11_CONTENT_TYPE) &&
+               !contentType.contains(SOAP12Constants.SOAP_12_CONTENT_TYPE);
     }
 
     static int populateSOAPVersion(MessageContext msgContext, String soapActionHeader, String contentType) {
```

Comment:
what does it mean by having soap version 0?

Reply:
Just a default value I guess. The user of the method just check whether the version is 1 or else. 
@menuka94 may be able to provide an explanation.